### PR TITLE
added how-to python to path on windows

### PR DESCRIPTION
**Pull request by valgaze**
- [x] documentation is changed or added
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Description of change

Adds procedure to set the path on Python on a "clean" windows10 installation (bootcamp)

### CODE REVIEW

**Comment by rvagg** [comment]
this looks good, aside from some nits ("environmental" and bad capitalisation) but I'd like confirmation from someone else that this is actually required when you use windows-build-tools, I haven't touched it myself

**Comment by rvagg** [comment]
@bzoz @joaocgreis would you mind taking a look at bc53cecd6b07f14c59480e2d51d4adb7d9d20610 and confirming that it's good to go, if so let's just merge it.

**Comment by bzoz** [comment]
https://github.com/nodejs/node-gyp/commit/bc53cecd6b07f14c59480e2d51d4adb7d9d20610 is not needed, windows-build-tools adds Python to the PATH environment variable itself.

File: gyp/pylib/gyp/common.py [hunk 1]
```diff
@@ -429,6 +429,10 @@ def GetFlavor(params):
     return 'netbsd'
   if sys.platform.startswith('aix'):
     return 'aix'
+  if sys.platform.startswith('zos'):
+    return 'zos'
+  if sys.platform.startswith('os390'):
+    return 'zos'
```
Comment (cclauss) [inline gyp/pylib/gyp/common.py]: https://docs.python.org/3/library/stdtypes.html#str.startswith Says __prefix can also be a tuple of prefixes to look for.__ so:
```python
if sys.platform.startswith(('os390', 'zos')):
    return 'zos'
```

**Comment by cclauss** [comment]
Tons of changes since the PR was opened.  Should it be rebased or closed? @rvagg @sam-github

**Comment by rvagg** [comment]
well since there's only one commit here and @bzoz' comment https://github.com/nodejs/node-gyp/pull/1356#issuecomment-505303952 suggests it's not relevant, so we close. @valgaze if you disagree feel free to dispute here but I guess it's been quite a while since you opened it!

**Pull request closed without merging**